appveyor.yml

```diff
@@ -1,11 +1,11 @@
-version: 1.0.{build}
+version: 2.0.{build}
```

Comment:
It would be nice for us to move to an auto-version system where Appveyor modifies the csproj's based on the tag pushed. e.g. https://github.com/Silvenga/Spectaris/blob/master/appveyor.yml#L5